source/core/coverage/proftab_temp_tables.sql

```diff
@@ -0,0 +1,233 @@
+create global temporary table plsql_profiler_runs$(
```

Comment:
OK. but then we do not clean up the profiler tables and they will continuously grow and will require clean up/maintenance by DBAs.